example/project-rails-tailwind/test/components/stories/input_component_stories.stories.json

```diff
@@ -0,0 +1,124 @@
+{
+  "title": "Atoms",
+  "stories": [
+    {
+      "name": "input",
+      "parameters": {
+        "server": {
+          "id": "input_component/input"
+        }
+      },
+      "args": {
+        "disabled": false,
```

Comment:
Looks like there are missing args for `label` and `hintText`. The component looks like it is set up to display them, but storybook doesn't give the user the option to set them.

Reply:
TY!